walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -391,4 +391,25 @@ public void noMandatoryAttendees2() {
 
         Assert.assertEquals(expected, actual);
     }
+
+    @Test
+    public void moreThanOneAttendee() {
```

Comment:
How about adding some comments to illustrate the test case with graphs, just as what the example test cases did.

Reply:
Test graphs are added! :)